shippable.yml

```diff
@@ -31,12 +35,24 @@ install:
   - ./hack/build-go.sh
   - GOPATH=$PWD/Godeps/_workspace:$GOPATH go install ./...
   - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH ./hack/verify-gendocs.sh
+  - curl -sL https://deb.nodesource.com/setup | sudo bash -
+  - sudo apt-get install -y nodejs firefox xvfb openjdk-7-jre-headless
+  - shippable_retry npm install karma@~0.10 karma-phantomjs-launcher@~0.1.4 karma-cli@0.0.4 karma-jasmine@^0.1.5 karma-junit-reporter@^0.2.2 karma-story-reporter@^0.3.1 protractor@^2.1.0
+  - ./node_modules/protractor/bin/webdriver-manager update
+
+before_script:
+  - export DISPLAY=:99.0
+  - XVFB=/usr/bin/Xvfb; XVFBARGS=":1 -screen 0 1024x768x24 -ac +extension GLX +render -noreset"; PIDFILE=/var/run/xvfb.pid;
+  - start-stop-daemon --start --quiet --pidfile $PIDFILE --make-pidfile --background --exec $XVFB -- $XVFBARGS
+  - sleep 3 # give xvfb some time to start
+  - nohup bash -c "./node_modules/protractor/bin/webdriver-manager start 2>&1 &"
+  - sleep 3 # give xvfb some time to start
 
 script:
-  - KUBE_RACE="-race" KUBE_COVER="y" KUBE_GOVERALLS_BIN="$HOME/gopath/bin/goveralls" KUBE_TIMEOUT='-timeout 300s' KUBE_COVERPROCS=8 KUBE_TEST_ETCD_PREFIXES="${KUBE_TEST_ETCD_PREFIXES}" KUBE_TEST_API_VERSIONS="${KUBE_TEST_API_VERSIONS}" ./hack/test-go.sh -- -p=2
-  - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH ./hack/test-cmd.sh
-  - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH KUBE_TEST_API_VERSIONS="${KUBE_TEST_API_VERSIONS}" KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4 LOG_LEVEL=4 ./hack/test-integration.sh
-  - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH ./hack/test-update-storage-objects.sh
+  # - KUBE_RACE="-race" KUBE_COVER="y" KUBE_GOVERALLS_BIN="$HOME/gopath/bin/goveralls" KUBE_TIMEOUT='-timeout 300s' KUBE_COVERPROCS=8 KUBE_TEST_ETCD_PREFIXES="${KUBE_TEST_ETCD_PREFIXES}" KUBE_TEST_API_VERSIONS="${KUBE_TEST_API_VERSIONS}" ./hack/test-go.sh -- -p=2
+  # - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH ./hack/test-cmd.sh
+  # - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH KUBE_TEST_API_VERSIONS="${KUBE_TEST_API_VERSIONS}" KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4 LOG_LEVEL=4 ./hack/test-integration.sh
```

Comment:
Oh, you're commenting out for testing? tag me here when you get it working & I'll review.